packages/angular/cli/src/commands/mcp/tools/projects.ts

```diff
@@ -15,6 +15,26 @@ import { AngularWorkspace } from '../../../utilities/config';
 import { assertIsError } from '../../../utilities/error';
 import { McpToolContext, declareTool } from './tool-registry';
 
+// Single source of truth for what constitutes a valid style language.
+const styleLanguageSchema = z.enum(['css', 'scss', 'sass', 'less']);
```

Comment:
Should we include tailwind here already or would that depend on support in the app creation flow first?

Reply:
Was going to handle tailwind separate since the heuristics are more complicated and i'm undecided on whether it should be represented here or in another form.